mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -543,10 +543,47 @@ func resourceContainerCluster() *schema.Resource {
 				ForceNew:    true,
 				Description: `Whether to enable Cloud TPU resources in this cluster.`,
 <% unless version == 'ga' -%>
-				Default: false,
+				ExactlyOneOf: []string{
+					"enable_tpu",
+					"tpu_config",
+				},
+				Computed: true,
```

Comment:
IIRC we consider this non-breaking as long as the API is currently returning false as the default?

Reply:
yes that's what I understand. The API doesn't return this field at all when not specified either with this field or `tpu_config`